src/app/people/people.component.ts

```diff
@@ -19,7 +19,9 @@ export class PeopleComponent implements OnInit {
   constructor(private HttpService: PeopleService) { }
```

Comment:
Why did you named `PeopleService ` as `HttpService `?

Reply:
Fixed